src/test/java/com/facebook/openwifirrm/ucentral/UCentralUtilsTest.java

```diff
@@ -17,4 +22,60 @@ public class UCentralUtilsTest {
 	void test_placeholder() throws Exception {
 		assertEquals(3, 1 + 2);
 	}
+
+	@Test
+	void test_setRadioConfigFieldChannel() throws Exception {
+		final String serialNumber = "aaaaaaaaaaaa";
+		final int expectedChannel = 10;
```

Comment:
Minor, maybe just a preference: maybe add `  // arbitrary` to the right of `10;`? Only because `10` is not a commonly used channel. Or change it to `1`?